gateway/src/apicast/policy/statuscode_overwrite/code_overwrite.lua

```diff
@@ -6,15 +6,14 @@ local new = _M.new
 function _M.new(config)
     local self = new(config)
     self.http_codes = {}
-    for _, code in ipairs(config.http_codes) do
+    for _, code in ipairs(config.http_codes or {}) do
```

Comment:
@eloycoto once you are revising it, isn'it better to have consistent variables name?
`http_codes --> http_statuses`

Reply:
Already updated on the jsonschema for the users ;-) 
https://github.com/3scale/APIcast/pull/1236/files#diff-d31ef981b73521817e16874af19636f5342d6791759acb1d77649d7eb2fff071R41

Reply:
I mean:
```
self.http_codes = {}
    for _, code in ipairs(config.http_codes or {}) do
```

to 
```
self.http_statuses = {}
    for _, code in ipairs(config.http_statuses or {}) do
```